code/modules/unit_tests/load_map_security.dm

```diff
@@ -0,0 +1,26 @@
+//Tests to ensure we can load a map from a whitelisted directory (_maps), but not a non-whitelisted directory (i.e "fartyShitPants")
+/datum/unit_test/load_map_security/Run()
+
+	//Replace runtimestation with another valid map .json if not present
+	var/goodMap = "runtimestation"
```

Comment:
Make this a define in this file instead of a var.

Reply:
Done - let me know if you want this in _defines/ somewhere - I based this off of other unit tests' use of defines